changelog/unreleased/pull-2671

```diff
@@ -0,0 +1,5 @@
+Feature: Addition of --min-packsize and --file-read-concurrency
```

Comment:
Judging from the changelog template https://github.com/restic/restic/blob/master/changelog/TEMPLATE this should read `Add [...] flags`